### ISSUE TITLE
Admin UI should require permissions

### ISSUE THREAD

**Issue by pgiraud** [GFDRR/thinkhazard#437]

**Comment by arnaud-morvan** [comment]
Could be a simple Apache BasicAuth configuration, exemple : 
```
<Location /amorvan/wsgi/admin>
    Order allow,deny
    Allow from all
    Require valid-user
    Satisfy All
    AuthType Basic
    AuthName "Authentication Required"
    AuthUserFile "/var/www/vhosts/wb-thinkhazard/conf/.htpasswd"
</Location>
```

**Comment by arnaud-morvan** [comment]
Another solution could be to use pyramid.security ACL.
User and password could be stored in settings file.
Here we have to set up : 
- a root object with some acl
- a user authentification/validation workflow
- a login form

**Comment by pgiraud** [comment]
Fixed via #446.
